packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -0,0 +1,827 @@
+	* @public
+	* @since 1.0.0-rc.11
+	*/
+	@property({ type: Object })
+	accessibilityTexts!: AccessiblilityTexts;
+
+	/**
+	* An object of strings that defines additional accessibility roles for further customization.
+	*
+	* It supports the following fields:
+	*  - <code>startColumnRole</code>: the accessibility role for the <code>startColumn</code>
+	*  - <code>startArrowContainerRole</code>: the accessibility role for the first arrow container (between the <code>begin</code> and <code>mid</code> columns)
+	*  - <code>midColumnRole</code>: the accessibility role for the <code>midColumn</code>
+	*  - <code>endArrowContainerRole</code>: the accessibility role for the second arrow container (between the <code>mid</code> and <code>end</code> columns)
+	*  - <code>endColumnRole</code>: the accessibility role for the <code>endColumn</code>
+	*
+	* @type {object}
+	* @public
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.accessibilityRoles
+	* @since 1.1.0
+	*/
+	@property({ type: Object })
+	accessibilityRoles!: AccessibilityRoles;
+
+	/**
+	* Defines the component width in px.
+	*
+	* @type {sap.ui.webc.base.types.Float}
+	* @defaultvalue 0
+	* @private
+	*/
+	@property({ validator: Float, defaultValue: 0 })
+	_width!: number;
+
+	/**
+	* Defines the effective columns layout,
+	* based on both the <code>layout</code> property and the screen size.
+	* Example: [67%, 33%, 0], [25%, 50%, 25%], etc.
+	*
+	* @type {object}
+	* @defaultvalue undefined
+	* @private
+	*/
+	@property({ type: Object, defaultValue: undefined })
+	_columnLayout?: ColumnLayout;
+
+	/**
+	* Defines the visible columns count - 1, 2 or 3.
+	*
+	* @type {sap.ui.webc.base.types.Integer}
+	* @defaultvalue 1
+	* @private
+	*/
+	@property({ validator: Integer, defaultValue: 0 })
+	_visibleColumns!: number;
+
+	/**
+	* Allows the user to replace the whole layouts configuration
+	*
+	* @type {object}
+	* @private
+	* @sap-restricted
+	*/
+	@property({ type: Object, defaultValue: undefined })
+	_layoutsConfiguration?: LayoutConfiguration;
+
+	/**
+	* Defines the content in the start column.
+	* @type {HTMLElement}
+	* @slot
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.startColumn
+	* @public
+	*/
+	@slot()
+	startColumn!: Array<HTMLElement>;
+
+	/**
+	* Defines the content in the middle column.
+	* @type {HTMLElement}
+	* @slot
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.midColumn
+	* @public
+	*/
+	@slot()
+	midColumn!: Array<HTMLElement>;
+
+	/**
+	* Defines the content in the end column.
+	* @type {HTMLElement}
+	* @slot
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.endColumn
+	* @public
+	*/
+	@slot()
+	endColumn!: Array<HTMLElement>;
+
+	initialRendering: boolean;
+	_handleResize: () => void;
+	static i18nBundle: I18nBundle;
+	_prevLayout: FCLLayout | null;
+
+	constructor() {
+		super();
+
+		this._prevLayout = null;
+		this.initialRendering = true;
+		this._handleResize = this.handleResize.bind(this);
```

Comment:
this._handleResize is bound in the constructor, do you need to bind here, is the eslint complaining

Reply:
Yes, the eslint is complaining.